src/dk/aau/cs/ds306e18/tournament/model/Team.java

```diff
@@ -16,7 +16,7 @@
 
     public Team(String teamName, ArrayList<Bot> bots, int initialSeedValue, String description) {
         this.teamName = teamName;
-        this.bots = bots;
+        this.bots = bots == null ? new ArrayList<>() : bots;
```

Comment:
> Why is null-behaviour handled here? Could the field bots not be initialised to an empty arraylist at line 11?
No, then it would just be overriden by whatever the constructor recieves. We have to make sure it is never set to null.

It can't be null, since there will be NullPointerExceptions. I will put a comment.